gulpfile.js

```diff
@@ -23,15 +23,15 @@ gulp.task('browser', function() {
     .pipe(gulp.dest('./Example/source'));
 });
 
-gulp.task('es6', function(){
+gulp.task('es6',_ => {
```

Comment:
@tarungarg546 Wouldn't it be better to ignore `browser` task if there is not `test.js` or `keys` folder?